### DISCUSSION TITLE
Add a button in explore view to move to prev/next time increment

### DISCUSSION THREAD

**Discussion by itsthatriver** [grafana/grafana#38956] [Explore]
In the metrics explorer if you have really spiky data it will render fine at certain display ranges (I've found 4 days seems like the threshold)
![image](https://user-images.githubusercontent.com/5632093/132452181-a45acb55-3bed-4cc5-a022-8fc9ef2f5246.png)
But if you zoom out you can lose a ton of information, making it hard to parse a graph quickly. This second screenshot is the exact same query on the exact same timeframe plus an additional day at the beginning
![image](https://user-images.githubusercontent.com/5632093/132452271-d17cdbc8-de7d-474f-b240-3ccb735cf836.png)

The pattern continues the further out you zoom. Overall this makes sense and I assume that selecting the most interesting data to display over a large timeframe with lots of data is hard. What would be helpful is if there were a button in the explore view so that you could find a useful zoom level (like 4 days) and then just page back through time. Right now the only way I've found to do this is to edit the time range by hand which is a clunky experience.

**Comment by itsthatriver** [comment]
After discovering `$__rate_interval` I no longer think this is necessary (would be nice to have, but the fidelity after zoomout problem is basically addressed already)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
